tests/testsuite/lints/unused_optional_dependencies.rs

```diff
@@ -221,7 +221,7 @@ Caused by:
 error: failed to parse manifest at `[ROOT]/foo/Cargo.toml`
 
 Caused by:
-  feature `feat` includes `dep?/feat`, but `dep` is not a dependency
+  feature `feat` includes `dep?/feat`, activate it in a feature with `dep:dep` if `dep` is an enabled dependency
```

Comment:
> if `dep` is an enabled dependency

I feel the wording could be improved here.  To help in coming up with ideas, could you expand on what you are trying to convey

Reply:
Yes, the messge need to include the cases that dependency inactive and dependency missing, `if dep is an enabled dependency` means the former.

https://github.com/rust-lang/cargo/pull/14026 is another PR to distinguish these. And this PR may require more detailed wording to address both cases.

Reply:
I suggest keep this PR to polish the message unless https://github.com/rust-lang/cargo/pull/14026https://github.com/rust-lang/cargo/pull/14026 is merged